Crucible.GameAPI/CrucibleIngredient.cs

```diff
@@ -416,6 +440,14 @@ public void SetStack(IEnumerable<CrucibleIngredientStackItem> rootItems)
             // It seems to remove itself automatically, as this component does not exist when inspecting the game object later.
             prefab.AddComponent<SortingOrderSetter>();
 
+            var slotObject = new GameObject
+            {
+                name = "Slot Object",
+            };
+            slotObject.transform.parent = prefab.transform;
+            slotObject.AddComponent<Slot>();
+            slotObject.AddComponent<ItemFromInventorySectionFinder>();
+
```

Comment:
Added basic slot components to prevent exceptions. Controller testing is needed as there are likely issues when using a controller. Currently StackItem's SlotObject has more components.